mithril-client/src/aggregator_client/certificate_client.rs

```diff
@@ -135,7 +135,9 @@ mod tests {
         http_client
             .expect_get_content()
             .return_once(move |_| {
-                Err(AggregatorHTTPClientError::RemoteServerLogical(String::new()))
+                Err(AggregatorHTTPClientError::RemoteServerLogical(anyhow!(
```

Comment:
Why not `String::new()` ?

Reply:
It refers to this comment from @jpraynaud: https://github.com/input-output-hk/mithril/pull/1250#discussion_r1334391825

Reply:
It looked a bit cumbersome to have the error spread over multiple lines, but this is still the case :man_shrugging: 
Maybe without the `to_string()` it could be better. 

Small detail IMHO, let's not waste too much energy on that :wink: